Packs/Wiz/IncidentTypes/incidenttype-WizIssue.json

```diff
@@ -0,0 +1,29 @@
+{
+    "id": "Wiz Issue",
+    "version": -1,
+    "vcShouldIgnore": false,
+    "locked": false,
+    "name": "Wiz Issue",
+    "prevName": "Wiz Issue",
+    "color": "#1084df",
+    "hours": 0,
+    "days": 0,
+    "weeks": 0,
+    "hoursR": 0,
+    "daysR": 0,
+    "weeksR": 0,
+    "system": false,
+    "readonly": false,
+    "default": false,
+    "autorun": false,
+    "disabled": false,
+    "reputationCalc": 0,
+    "onChangeRepAlg": 1,
+    "layout": "1fe99b8a-f4b5-4214-80bc-2f561a71d9ed",
```

Comment:
I believe the value should be "Wiz Layout"

Reply:
Yes, there is a commit coming in a bit to fix it.

Reply:
Added in 58490dc83597062f51291b7ecd158d30fd9f6034